src/open_and_save.c

```diff
@@ -53,8 +52,6 @@ void read_lines(void) {
         lines[0].length = 0;
         lines[0].data[0] = '\0';
         lines[0].ident = 0;
-        
-        syntaxHighlight();
```

Comment:
Why :?

Reply:
The line is blank, and when we write any printable character (or newline) syntaxHighlight will be called

Reply:
That is also the only line in the file